libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Skills/SkillConversationIdFactory.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Skills;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Integration.Runtime.Skills
+{
+    /// <summary>
+    /// A <see cref="SkillConversationIdFactory"/> that uses <see cref="IStorage"/> to store
+    /// and retrieve <see cref="SkillConversationReference"/> instances.
+    /// </summary>
+    internal class SkillConversationIdFactory : SkillConversationIdFactoryBase
```

Comment:
similar comment to above - why isn't this code in Microsoft.Bot.Builder?

Reply:
The aim of this PR is to be additive only. When this becomes GA, the pieces that make sense to move, will be moved to GA assemblies. For example, the plugin assembly will not exist, but it exists to allow us to preview that before it moves to Microsoft.Bot.Builder. This would go into the Integration.AspNetCore assembly in R13 as part of getting this to GA.

Reply:
#wontfix until GA